Client/WebFilters/FilterListResourceDownloader.swift

```diff
@@ -111,6 +111,16 @@ public class FilterListResourceDownloader: ObservableObject {
       allFilterListSettings.append(setting)
     }
   }
+  
+  /// The uuid of the cookie consent notices filter list.
+  ///
+  /// This is a special filter list that has more accessible UI to control it
+  public static let cookieConsentNoticesUUID = "AC023D22-AE88-4060-A978-4FEEEC4221693"
+  /// The uuid of the "Remove Web annoyances" filter list
+  ///
+  /// This is a special filter list that has more accessible UI to control it
+  public static let removeWebAnnoyancesUUID = "67E792D4-AE03-4D1A-9EDE-80E01C81F9B8"
```

Comment:
we can use `componentID` if possible, apparently the UUIDs are going to be deprecated

Reply:
I can't remove all references of UUID yet but I believe here it will be ok.